webapp/package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "build": "webpack --mode=production",
     "build:watch": "webpack --mode=production --watch",
-    "debug": "webpack --mode=none",
+    "debug": "webpack --mode=production",
```

Comment:
On debug we want production mode? If so, wouldn't we want also production mode in `debug:watch`? 0/5

Reply:
Ah no, this should be `development`.